packages/server/lib/utils/tests.ts

```diff
@@ -7,20 +7,29 @@ import { getServerPort } from '@nangohq/shared';
 
 import { app } from '../routes.js';
 
+function uriParamsReplacer(tpl: string, data: Record<string, any>) {
+    return tpl.replace(/:([a-zA-Z]+)/g, function (_, name) {
+        return data[name];
+    });
```

Comment:
if we want to avoid regexes 
```
function uriParamsReplacer(tpl: string, data: Record<string, any>) {
            let res = tpl
            for (const [key, value] of Object.entries(data)) {
                res = res.replace(`:${key}`, value);
            }
            return res;
});
```

Reply:
true, it's more readable